### ISSUE TITLE
Dreamhost's default cache service is not working with Pods.

### ISSUE THREAD

**Issue by aditi-Kh** [pods-framework/pods#5785]
## Issue Overview
Dreamhost's default cache service is not working with Pods.

## Expected Behavior
Caching should happen even if the pods plugin is active

## Current Behavior
Caching stops working if pods plugin is activated and the website becomes extremely slow

## Steps to Reproduce (for bugs)
1. Website was slow and cache was not working. So, ran purge cache check and it gave pods as the blocker for cache not working.
2. Deactivated Pods 
3. Ran the Purge Cache check again. 
4. Website was caching now.
5. Activated Pods again
6. Ran purge cache check and it gave pods as the blocker for cache not working again.

## WordPress Environment
WordPress Version: 5.4.2
<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
Debug Information
WordPress Version: 5.4.2

PHP Version: 7.4.3

MySQL Version: 5.7.29

Server Software: Apache

Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36

Session Save Path: /tmp

Session Save Path Exists: Yes

Session Save Path Writeable: Yes

Session Max Lifetime: 1440

Opcode Cache:

Apc: No
Memcached: No
OPcache: Yes
Redis: No
Object Cache:

APC: No
APCu: No
Memcache: No
Memcached: No
Redis: No
WPDB Prefix: tetrate_

WP Multisite Mode: No

WP Memory Limit: 40M

Current Memory Usage: 39.936M

Current Memory Usage (real): 8.000M

Pods Network-Wide Activated: No

Pods Install Location: /home/wp_qdsu98/tetrate.io/wp-content/plugins/pods/

Pods Tableless Mode Activated: No

Pods Light Mode Activated: No

Currently Active Theme: AmberSix

Currently Active Plugins:

Carousel Anything - VC Extension: 2.0
Classic Editor: 1.5
DreamHost Panel Login: 1.0.0
Duplicate Page: 4.2
Easy Updates Manager: 9.0.3
Events Shortcodes and Templates Addon: 1.5.3
Google Analytics for WordPress by MonsterInsights: 7.11.0
HubSpot All-In-One Marketing - Forms, Popups, Live Chat: 7.36.1
Loginizer: 1.4.8
Lordicon Interactive Icons: 1.6.0
Meta Box: 5.3.2
Pods - Custom Content Types and Fields: 2.7.21
Proxy Cache Purge: 4.8.1
Simple 301 Redirects: 1.07
Simple History: 2.34.0
Slider Revolution: 6.2.2
The Events Calendar: 5.1.4
WP Anything Downloader: 3.0.1
WPBakery Page Builder: 6.2.0
WP Gatsby: 0.4.12
WP GraphQL: 0.10.3
WPRT Addons: 3.7.9
Yoast SEO: 14.5
## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```
{"meta":{"version":"2.7.21","build":1595224648},"pods":{"23321":{"id":23321,"name":"event","label":"Events","description":"","type":"post_type","storage":"meta","object":"","alias":"","fields":{"time_date":{"id":23325,"name":"time_date","label":"Time & Date","description":"","help":"","class":"","type":"datetime","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"1","datetime_type":"format","datetime_format":"mdy","datetime_time_type":"12","datetime_time_format":"h_mma","datetime_time_format_24":"hh_mm","datetime_allow_empty":"1","datetime_html5":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","datetime_repeatable":"0","datetime_format_custom":"","datetime_format_custom_js":"","datetime_time_format_custom":"","datetime_time_format_custom_js":"","datetime_year_range_custom":""},"venue":{"id":23326,"name":"venue","label":"Venue","description":"","help":"","class":"","type":"text","weight":1,"pick_object":"","pick_val":"","sister_id":"","required":"0","text_allow_shortcode":"0","text_allow_html":"1","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","text_repeatable":"0","text_placeholder":""},"event_website":{"id":23327,"name":"event_website","label":"Event Website","description":"","help":"","class":"","type":"website","weight":2,"pick_object":"","pick_val":"","sister_id":"","required":"0","website_format":"normal","website_allow_port":"0","website_clickable":"0","website_new_window":"0","website_max_length":"255","website_html5":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","website_repeatable":"0","website_placeholder":""},"organizers":{"id":23328,"name":"organizers","label":"Organizers","description":"","help":"","class":"","type":"oembed","weight":3,"pick_object":"","pick_val":"","sister_id":"","required":"0","oembed_width":"0","oembed_height":"0","oembed_show_preview":"0","oembed_restrict_providers":"0","oembed_enabled_providers_amazoncn":"0","oembed_enabled_providers_amazoncouk":"0","oembed_enabled_providers_amazoncom":"0","oembed_enabled_providers_amazoncomau":"0","oembed_enabled_providers_amazonin":"0","oembed_enabled_providers_animotocom":"0","oembed_enabled_providers_cloudupcom":"0","oembed_enabled_providers_crowdsignalcom":"0","oembed_enabled_providers_dailymotioncom":"0","oembed_enabled_providers_facebookcom":"0","oembed_enabled_providers_flickrcom":"0","oembed_enabled_providers_hulucom":"0","oembed_enabled_providers_imgurcom":"0","oembed_enabled_providers_instagramcom":"0","oembed_enabled_providers_issuucom":"0","oembed_enabled_providers_kickstartercom":"0","oembed_enabled_providers_meetupcom":"0","oembed_enabled_providers_mixcloudcom":"0","oembed_enabled_providers_redditcom":"0","oembed_enabled_providers_reverbnationcom":"0","oembed_enabled_providers_screencastcom":"0","oembed_enabled_providers_scribdcom":"0","oembed_enabled_providers_slidesharenet":"0","oembed_enabled_providers_smugmugcom":"0","oembed_enabled_providers_someecardscom":"0","oembed_enabled_providers_soundcloudcom":"0","oembed_enabled_providers_speakerdeckcom":"0","oembed_enabled_providers_spotifycom":"0","oembed_enabled_providers_tedcom":"0","oembed_enabled_providers_tiktokcom":"0","oembed_enabled_providers_tumblrcom":"0","oembed_enabled_providers_twittercom":"0","oembed_enabled_providers_vimeocom":"0","oembed_enabled_providers_wordpresscom":"0","oembed_enabled_providers_wordpresstv":"0","oembed_enabled_providers_youtubecom":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","oembed_repeatable":"0"}},"show_in_menu":"1","label_singular":"Event","public":"1","show_ui":"1","supports_title":"1","supports_editor":"1","publicly_queryable":"1","exclude_from_search":"0","capability_type":"post","capability_type_custom":"event","capability_type_extra":"1","has_archive":"0","hierarchical":"0","rewrite":"1","rewrite_with_front":"1","rewrite_feeds":"0","rewrite_pages":"1","query_var":"1","can_export":"1","default_status":"draft","supports_author":"0","supports_thumbnail":"1","supports_excerpt":"0","supports_trackbacks":"0","supports_custom_fields":"0","supports_comments":"0","supports_revisions":"1","supports_page_attributes":"1","supports_post_formats":"0","built_in_taxonomies_category":"0","built_in_taxonomies_link_category":"0","built_in_taxonomies_member_category":"0","built_in_taxonomies_partner_category":"0","built_in_taxonomies_post_tag":"0","built_in_taxonomies_project_category":"0","built_in_taxonomies_testimonials_category":"0","built_in_taxonomies_tribe_events_cat":"1","menu_position":"0","show_in_nav_menus":"1","show_in_admin_bar":"1","pfat_enable":"0","pfat_run_outside_loop":"0","pfat_single":"Job Openings","pfat_append_single":"append","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"1","read_all":"1","write_all":"0"},"22820":{"id":22820,"name":"job_listing","label":"Job Listings","description":"Job Listings at Tetrate","type":"post_type","storage":"meta","object":"","alias":"","fields":{"job_role":{"id":22821,"name":"job_role","label":"Job Role","description":"Role which we are hiring for","help":"","class":"","type":"text","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"1","text_allow_shortcode":"1","text_allow_html":"1","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"1","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","unique":"0","text_repeatable":"0","text_placeholder":""}},"show_in_menu":"1","label_singular":"Job Listing","public":"1","show_ui":"1","supports_title":"1","supports_editor":"1","publicly_queryable":"1","exclude_from_search":"0","capability_type":"post","capability_type_custom":"job_listing","capability_type_extra":"1","has_archive":"0","hierarchical":"0","rewrite":"1","rewrite_custom_slug":"careers","rewrite_with_front":"0","rewrite_feeds":"0","rewrite_pages":"1","query_var":"1","can_export":"1","default_status":"draft","supports_author":"1","supports_thumbnail":"0","supports_excerpt":"0","supports_trackbacks":"0","supports_custom_fields":"0","supports_comments":"0","supports_revisions":"0","supports_page_attributes":"1","supports_post_formats":"0","built_in_taxonomies_category":"1","built_in_taxonomies_link_category":"0","built_in_taxonomies_member_category":"0","built_in_taxonomies_partner_category":"0","built_in_taxonomies_post_tag":"0","built_in_taxonomies_project_category":"0","built_in_taxonomies_testimonials_category":"0","built_in_taxonomies_tribe_events_cat":"0","menu_position":"0","show_in_nav_menus":"1","show_in_admin_bar":"1","pfat_enable":"1","pfat_run_outside_loop":"1","pfat_append_single":"append","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"1","read_all":"1","write_all":"0","pfat_single":"Job Openings"},"22474":{"id":22474,"name":"project","label":"Project","description":"","type":"post_type","storage":"meta","object":"project","alias":"","fields":[],"show_in_menu":1,"built_in_taxonomies_event":"0"},"22475":{"id":22475,"name":"resource","label":"Resources","description":"","type":"post_type","storage":"meta","object":"","alias":"","fields":{"resource_title":{"id":22850,"name":"resource_title","label":"Resource Title","description":"About the resource details","help":"","class":"","type":"text","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"0","text_allow_shortcode":"0","text_allow_html":"0","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","text_repeatable":0,"text_placeholder":""}},"show_in_menu":"1","label_singular":"Resource","public":"1","show_ui":"1","supports_title":"1","supports_editor":"1","publicly_queryable":"1","exclude_from_search":"0","capability_type":"post","capability_type_custom":"resource","capability_type_extra":"1","has_archive":"0","hierarchical":"0","rewrite":"1","rewrite_with_front":"1","rewrite_feeds":"0","rewrite_pages":"1","query_var":"1","can_export":"1","default_status":"draft","supports_author":"0","supports_thumbnail":"1","supports_excerpt":"0","supports_trackbacks":"0","supports_custom_fields":"1","supports_comments":"0","supports_revisions":"0","supports_page_attributes":"1","supports_post_formats":"1","built_in_taxonomies_category":"1","built_in_taxonomies_link_category":"0","built_in_taxonomies_member_category":"0","built_in_taxonomies_partner_category":"0","built_in_taxonomies_post_tag":"1","built_in_taxonomies_project_category":"0","built_in_taxonomies_testimonials_category":"0","built_in_taxonomies_tribe_events_cat":"0","menu_position":"0","show_in_nav_menus":"1","show_in_admin_bar":"1","pfat_enable":"1","pfat_run_outside_loop":"1","pfat_append_single":"append","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"1","read_all":"1","write_all":"0"},"23320":{"id":23320,"name":"tribe_events","label":"Tribe Events","description":"","type":"post_type","storage":"meta","object":"tribe_events","alias":"","fields":[],"show_in_menu":1}},"templates":{"Job Openings":{"id":22883,"name":"Job Openings","code":"<p><a href=\"{@permalink}\" class=\"careers-link\"><span>{@job_role}<\/span><\/a><\/p>","type":"template","slug":"job-openings","options":{"slide_template":"","rs_page_bg_color":"","_edit_last":"3","admin_only":"","restrict_capability":"","capability_allowed":"","_edit_lock":"1595169334:1","pod_reference":"a:1:{s:3:\"pod\";s:11:\"job_listing\";}"}},"Job Description":{"id":22864,"name":"Job Description","code":"","type":"template","slug":"job-post","options":{"_edit_last":"3","slide_template":"","rs_page_bg_color":"","admin_only":"","restrict_capability":"","capability_allowed":"","_edit_lock":"1595166050:1","pod_reference":"a:1:{s:3:\"pod\";s:11:\"job_listing\";}"}}}}

**Comment by sc0ttkclark** [comment]
Have you tried turning sessions off for Pods? Try this in your wp-config.php:

`define( 'PODS_SESSION_AUTO_START', false );`

**Comment by sc0ttkclark** [comment]
Sessions have been addressed in Pods 2.8 🎉
